compsets/fv3mom6cice5_ccpp.input

```diff
@@ -387,6 +401,187 @@ test cpld_fv3_ccpp_mom6_cice_1d_satmedmf: fv3_ccpp_mom6_cice.exe {
+        'RESTART/fv_tracer.res.tile3.nc'        .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/fv_tracer.res.tile4.nc'        .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/fv_tracer.res.tile5.nc'        .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/fv_tracer.res.tile6.nc'        .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/phy_data.tile1.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/phy_data.tile2.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/phy_data.tile3.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/phy_data.tile4.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/phy_data.tile5.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/phy_data.tile6.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/sfc_data.tile1.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/sfc_data.tile2.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/sfc_data.tile3.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/sfc_data.tile4.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/sfc_data.tile5.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'RESTART/sfc_data.tile6.nc'             .bitcmp. "@[CNTL]/RESTART/"
+        'mediator_FBAtm_a_restart.tile1.nc'     .bitcmp. "@[CNTLMED]/"
+        'mediator_FBAtm_a_restart.tile2.nc'     .bitcmp. "@[CNTLMED]/"
+        'mediator_FBAtm_a_restart.tile3.nc'     .bitcmp. "@[CNTLMED]/"
+        'mediator_FBAtm_a_restart.tile4.nc'     .bitcmp. "@[CNTLMED]/"
+        'mediator_FBAtm_a_restart.tile5.nc'     .bitcmp. "@[CNTLMED]/"
+        'mediator_FBAtm_a_restart.tile6.nc'     .bitcmp. "@[CNTLMED]/"
+        'mediator_FBAtmOcn_o_restart.nc'        .bitcmp. "@[CNTLMED]/"
+        'mediator_FBIce_i_restart.nc'     .bitcmp. "@[CNTLMED]/"
+        'mediator_FBOcn_o_restart.nc'     .bitcmp. "@[CNTLMED]/"
+        'mediator_scalars_restart.txt'    .bitcmp. "@[CNTLMED]/"
+      # Executable validation.  This makes an MD5 sum of the fv3.exe
+      # for comparison against the MD5 sum made in the build job.
+      # This is to ensure the executable did not change during the
+      # test suite.
+        "@[build%target]" .md5cmp. "@[fv3_ccpp_mom6_cice_debug.exe%md5sum]" 
+    }
+} 
+######################################################################## 
+# warm 6h c96 debug
+
+test cpld_fv3_ccpp_mom6_cice_6h_debug: fv3_ccpp_mom6_cice_debug.exe { 
+    use cpld_fv3_ccpp_mom6_cice_2d_atm_flux 
+    DAYS='0.25'  # run for 0.25 day
+    FHMAX='6'
+    NFHOUT_HF='1'
+    walltime=2400   # seconds
+    TEST_DESCR="Fully coupled FV3-CCPP-MOM6-CICE system - debug"
+    CNTL_NAME="RT-Baselines_6h_warm_debug_ccpp"
+    CNTL="@[plat%BASELINE]/@[CNTL_NAME]" 
+
+    MED_restart_data="@[plat%BASELINE]/MEDIATOR_debug_ccpp"
+
+    build=fv3_ccpp_mom6_cice_debug.exe
+
+    prep=fv3cpld_prep(
+    RUNDIR="@[RUNDIR]",modules="@[fv3_ccpp_mom6_cice_debug.exe%modules.nems]",
+    CNTL="@[CNTL]")
+
+    # Specify output files:
```

Comment:
You are not comparing the CICE restart file. I believe that is because you have not created one because the default compset you are using has: 
 # write a CICE restart at DAYS
    DUMPFREQ_N="@[DAYS]"
    DUMPFREQ='d'

In order to generate a CICE restart for a 6 hour run, you need to set 
DUMPFREQ_N ="@[FHMAX]" 
DUMPFREQ='h'

Reply:
You are correct. The cice_in will be updated in the next commit this week.

Reply:
So we will update the test to produce the ice restart file and also compare it, correct?